.goreleaser.yml

```diff
@@ -2,11 +2,16 @@
 # Make sure to check the documentation at http://goreleaser.com
 before:
   hooks:
-    # you may remove this if you don't use vgo
-    - go mod download
+  - go mod vendor
 builds:
 - env:
   - CGO_ENABLED=0
+  - GOROOT=/usr/local/go
+  ignore:
```

Comment:
Do we need to maintain 386 compat? if we deploy to 32 bit systems i'd rather this be isolated to ARMv6/ARMv7 and skip 386 all together. Just so its easier to test/maintain on systems we're actively using.

Reply:
i'd say we prolly dont need 386 compat

if we ever end up needing it in the future, it'd be easy enough to enable